levels/castle_inside/clock_minute_hand/geo.inc.c

```diff
@@ -1,6 +1,6 @@
 // 0x0E001530
 const GeoLayout castle_geo_001530[] = {
-   GEO_CULLING_RADIUS(300),
+   GEO_CULLING_RADIUS(500),
```

Comment:
Same here

Reply:
Yes, without it, the clock hands get culled at the very edge of the BitDW hallway (and most likely the other end of the castle courtyard)